rivertile/main.zig

```diff
@@ -88,6 +88,10 @@ var default_main_ratio: f64 = 0.6;
 /// We don't free resources on exit, only when output globals are removed.
 const gpa = std.heap.c_allocator;
 
+/// Arbitrary minimal view size. This allows us to assert a minimum width and
+/// height, making the layout math nicer.
+const min_view_size = 10;
```

Comment:
It would be a lot simpler to just do `ev.usable_width -| 2 *outer_padding` IMO. I kinda hate the constants like this that we already have and would like to get rid of them.

Reply:
I'm not a fan of the constants either, however `ev.usable_width -| 2 * outer_padding` would still allow a usable width/height of `0`, which is nonsensical. In theory we should be able to rely on clients to have a sane minimum view size, but in practice this is not the case, so we probably need an arbitrary minimum view size  constant *somewhere*. And it does remove an edge case. However I do think river is the better place to enforce this than rivertile, so I'll update this patch accordingly.